azure-dataplane-tests/src/main/java/com/azure/ai/formrecognizer/documentanalysis/implementation/models/DocumentModelComposeOperationDetails.java

```diff
@@ -146,6 +116,36 @@ public DocumentModelComposeOperationDetails setError(Error error) {
         return this;
     }
 
+    /**
+     * Get the kind property: Type of operation.
+     * 
+     * @return the kind value.
+     */
+    @Override
+    public String getKind() {
+        return this.kind;
+    }
+
+    /**
+     * Get the result property: Operation result upon success.
+     * 
+     * @return the result value.
+     */
+    public DocumentModelDetails getResult() {
+        return this.result;
+    }
+
+    /**
+     * Set the result property: Operation result upon success.
+     * 
+     * @param result the result value to set.
```

Comment:
Is it just changing the order?

Reply:
It is just a change in order but it's a good time to have a discussion on how we want to order overridden methods. Before this change, the methods defined the type came first then overridden methods, but in serialization code the properties defined by super types came first. It'd be nice to have a standard system for both serialization and getters/setters, such as properties and setters/getters defined by super types always come first (or last).

Reply:
To me, as long as the two orders (methods order and properties processing order during serialziation) are consistent, and both override methods and new methods have their own section, that would be fine.
I also asked copilot. It says override methods from super class comes first. So I would vote for this - override methods come first. :)
<img width="371" alt="image" src="https://github.com/user-attachments/assets/37ee4364-2491-4c10-9f4c-3d150816147b">